package.json

```diff
@@ -72,7 +72,7 @@
     "@open-wc/testing": "4.0.0",
     "@rollup/plugin-babel": "6.0.4",
     "@rollup/plugin-node-resolve": "15.2.3",
-    "@rollup/plugin-replace": "5.0.7",
+    "@rollup/plugin-replace": "6.0.1",
```

Comment:
Branch was built from main so this is expected when merging into `release-20240924`.